server/grpc_service.go

```diff
@@ -1306,10 +1306,9 @@ func (s *Server) SyncMaxTS(ctx context.Context, request *pdpb.SyncMaxTSRequest)
 	if err != nil {
 		return nil, status.Errorf(codes.Unknown, err.Error())
 	}
-	var processedDCs []string
-	if request.GetMaxTs() == nil || request.GetMaxTs().GetPhysical() == 0 {
-		// The first phase of synchronization: collect the max local ts
-		var maxLocalTS pdpb.Timestamp
+	var maxLocalTS pdpb.Timestamp
+	if !request.GetSkipCheck() {
```

Comment:
The original logic need 2 phase syncing to get global tso:
1. collect maxTS from each local tso allocator
2. write maxTSO to each local tso allocator

The optimized logic only need 1 phase syncing to get global tso:
1. estimate the  maxTSO locally
2.  write maxTSO to each tso allocator.

So I think there shouldn't be code change in `SyncMaxTS`